.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  - push
+  - pull_request
```

Comment:
Any idea why this didn't run here?

Reply:
It looks like the actions are executing on the [forked repo](https://github.com/somleng/adhearsion/actions/runs/1879253934). Strange that they do not appear here as well.

Reply:
This repo has the correct configuration. I don't think these manifests need to be on the main branch for it to run.

Reply:
<img width="1118" alt="Screenshot 2022-03-01 at 09 51 37" src="https://user-images.githubusercontent.com/210221/156172187-f2eecda4-61d1-407c-9b58-b7ce7fe6be39.png">

Reply:
Because @dwilkie is not a first time contributor. It should run without approval.

Reply:
Not even setting "Require approval for all outside collaborators" makes the button show up, though.

Reply:
I can't see any reason these actions aren't running. Since Travis is already broken, any objection to merging this PR as-is and see if that's what kickstarts Github into running the actions?

Reply:
If we're all out of ideas, I guess that's all we can do :)